frontends/concrete-python/Makefile

```diff
@@ -173,7 +173,12 @@ pylint:
 
 ruff:
 	eval $(shell make silent_cp_activate)
-	ruff concrete/ examples/ scripts/ tests/ benchmarks/
+
+	ruff check concrete/
```

Comment:
I can restore on a single line if you want

Reply:
no, the old `ruff concrete/ examples/ scripts/ tests/ benchmarks/` is no more supported, they explicitly request to move to `ruff check`

Reply:
Ah okay, this is fine then!